src/AggregationFunction/FunctionSum.php

```diff
@@ -53,6 +59,16 @@ public function processDataSource($data_source)
 				->fetch()
 				->sum;
 		}
+		if ($data_source instanceof \Doctrine\ORM\QueryBuilder) {
+			$data_source_clone = clone $data_source;
```

Comment:
I think `$queryBuilder` should be cloned in the DoctrineDataSource itself as it should keep its internal data consistent and thus make sure it does not expose them.